tests/foreman/cli/test_contenthost.py

```diff
@@ -731,3 +733,45 @@ def test_contenthost_package_upgrade_all(self):
         self.assertEqual(result.return_code, 0)
         result = self.vm.run('rpm -q {}'.format(FAKE_2_CUSTOM_PACKAGE))
         self.assertEqual(result.return_code, 0)
+
+    def test_contenthost_package_group_install(self):
+        """@Test: Install package group to content host remotely
+
+        @Feature: Content Host - Package group
+
+        @Assert: Package group was successfully installed
+
+        """
+        result = ContentHost.package_group_install({
+            u'organization-id': TestCHKatelloAgent.org['id'],
+            u'content-host': self.vm.hostname,
+            u'groups': FAKE_0_CUSTOM_PACKAGE_GROUP_NAME,
+        })
+        self.assertEqual(result.return_code, 0)
+        for package in FAKE_0_CUSTOM_PACKAGE_GROUP:
+            result = self.vm.run('rpm -q {}'.format(package))
+            self.assertEqual(result.return_code, 0)
+
+    def test_contenthost_package_group_remove(self):
+        """@Test: Remove package group from content host remotely
+
+        @Feature: Content Host - Package group
+
+        @Assert: Package group was successfully removed
+
+        """
+        result = ContentHost.package_group_install({
+            u'organization-id': TestCHKatelloAgent.org['id'],
+            u'content-host': self.vm.hostname,
+            u'groups': FAKE_0_CUSTOM_PACKAGE_GROUP_NAME,
+        })
+        self.assertEqual(result.return_code, 0)
+        result = ContentHost.package_group_remove({
+            u'organization-id': TestCHKatelloAgent.org['id'],
+            u'content-host': self.vm.hostname,
+            u'groups': FAKE_0_CUSTOM_PACKAGE_GROUP_NAME,
+        })
+        self.assertEqual(result.return_code, 0)
```

Comment:
First approach looks better for me, but yea, it's a little bit tricky. Ok, will update with the second approach then :)